hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -179,9 +179,9 @@
                 "rds:CreateDBInstanceReadReplica"
             ],
             "Resource": [
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:snapshot-*",
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:rds-*",
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:db:rds-*"
+                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:ansible-test*",
```

Comment:
Partial wildcards work _sometimes_ and completely randomly.  My technique in building these rules has been to start by blocking everything, then iteratively 1) cure the exceptions so  that they give me the information I need, 2) open up the rule the minimum amount so that it might fix the problem 3) test again.  4) goto 1.  This means that since I know that the tests are passing in my environment I'm reasonably sure the IAM rules work.